app/views/prior_authority/applications/state_summaries/_expired.html.erb

```diff
@@ -0,0 +1,4 @@
+<p><strong class="govuk-tag <%= prior_authority_tag_colour(:rejected) %>"><%= t(".status") %></strong></p>
```

Comment:
Although the colour is the same we should probably have a separate tag colour in the helper for clarity
```suggestion
<p><strong class="govuk-tag <%= prior_authority_tag_colour(:expired) %>"><%= t(".status") %></strong></p>
```